ssl/statem/statem_clnt.c

```diff
@@ -1432,7 +1432,7 @@ static int set_client_ciphersuite(SSL_CONNECTION *s,
              * In TLSv1.3 it is valid for the server to select a different
              * ciphersuite as long as the hash is the same.
              */
-            if (md == NULL
+            if (s->session->cipher == NULL || md == NULL
```

Comment:
Is it already checked above through line 1427 to 1430?

Reply:
Yes - so this PR is not needed.